### PR TITLE
Revert 5 dependabot/npm and yarn/node fetch 2.6.1

### PR DESCRIPTION
**Pull request by Kapp1**
Closes #[issue number]

## Description

-

## Release notes

Notes:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # [GitHub Desktop](https://desktop.github.com)
 
-## Hello Universe!!
+## Hello Universe!!!
```
Comment (Kapp1) [inline README.md:3]:
```suggestion
## Hello Universe!!!
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # [GitHub Desktop](https://desktop.github.com)
 
+## Hello Universe!!!
+
 [![Travis](https://img.shields.io/travis/desktop/desktop.svg?style=flat-square&label=Travis+CI)](https://travis-ci.org/desktop/desktop)
 [![CircleCI](https://img.shields.io/circleci/project/github/desktop/desktop.svg?style=flat-square&label=CircleCI)](https://circleci.com/gh/desktop/desktop)
 [![AppVeyor Build Status](https://img.shields.io/appveyor/ci/github-windows/desktop/development.svg?style=flat-square&label=AppVeyor&logo=appveyor)](https://ci.appveyor.com/project/github-windows/desktop/branch/development)
```
Comment (Kapp1) [inline README.md:7]:
```suggestion
[![AppVeyor Build Status](https://img.shields.io/appveyor/ci/github-windows/desktop/development.svg?style=flat-square&label=AppVeyor&logo=appveyor)](https://ci.appveyor.com/project/github-windows/desktop/branch/development)
```

**Comment by niik** [comment]
Thanks for your submission.

It appears that you've created a pull request that doesn't add functionality to the GitHub Desktop project so we're going to go ahead and close this. If it was intentional, please let us know what you were intending and we can see about reopening it.

Thanks again!

**Comment by Kapp1** [comment]
Thank you I just test

**Pull request closed without merging**